File: examples/classes.swift
typealias TestBool = Bool

typealias TestByte = Int

typealias TestBytes = Array<Int>

typealias TestChan = Any

typealias TestFloat32 = Float

typealias TestFloat64 = Double

typealias TestFunc = Any

typealias TestInt = Int

typealias TestInt16 = Int

typealias TestInt32 = Int

typealias TestInt64 = Int

typealias TestInt8 = Int

typealias TestInterface = Dictionary<String, Any>

typealias TestMap = Dictionary<String, String>

typealias TestMapInt = Dictionary<Int, String>

typealias TestPointer = String?

class TestPointerStruct {
	PP String?;
}

typealias TestRune = Int

typealias TestSliceSlice = Array<TestSliceString>

typealias TestSliceString = Array<String>

typealias TestString = String

class TestStruct1 {
	One Dictionary<String, String>;
}

class TestStruct2Named {
	Two String;
}

class TestStruct3Extend: TestStruct1 {
	TestStruct2Named TestStruct2Named;
}

class TestStruct5UsePackage {
	ValNotExp testStruct4NotExported;
}

typealias TestUint = Int

typealias TestUint16 = Int

typealias TestUint32 = Int

typealias TestUint64 = Int

typealias TestUint8 = Int

typealias TestUintptr = Any

class testStruct4NotExported {
	ExportMe Bool;
}

File: examples/classes.dart
typedef TestBool = bool

typedef TestByte = int

typedef TestBytes = List<int>

typedef TestChan = dynamic

typedef TestFloat32 = double

typedef TestFloat64 = double

typedef TestFunc = dynamic

typedef TestInt = int

typedef TestInt16 = int

typedef TestInt32 = int

typedef TestInt64 = int

typedef TestInt8 = int

typedef TestInterface = Map<String, dynamic>

typedef TestMap = Map<String, String>

typedef TestMapInt = Map<int, String>

typedef TestPointer = String?

class TestPointerStruct {
	PP String?;
}

typedef TestRune = int

typedef TestSliceSlice = List<TestSliceString>

typedef TestSliceString = List<String>

typedef TestString = String

class TestStruct1 {
	One Map<String, String>;
}

class TestStruct2Named {
	two String;
}

class TestStruct3Extend extends TestStruct1 {
	TestStruct2Named TestStruct2Named;
}

class TestStruct5UsePackage {
	notExp testStruct4NotExported;
}

class TestStruct6NamedOmitempty {
	one String?;
}

class TestStruct7NamedOmit {
}

typedef TestUint = int

typedef TestUint16 = int

typedef TestUint32 = int

typedef TestUint64 = int

typedef TestUint8 = int

typedef TestUintptr = dynamic

class testStruct4NotExported {
	exportMe bool;
}

